testing/btest/Baseline/scripts.base.protocols.dns.event-handler-warning/.stderr

```diff
@@ -0,0 +1,6 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+warning in <...>/check-event-handlers.zeek, line 15: Used event 'dns_EDNS_tcp_keepalive' will not be raised because 'dns_skip_all_addl' is true
```

Comment:
I think for making noise this is good. I might have tried in C++ `InitPostScript()` in the DNS analyzer which would've allowed to get the locations of the actual handlers, but that'd be for polishing the UX. Script-land is also nice.